### PR TITLE
Fix build in debug mode

### PR DESCRIPTION
**Pull request by dimitry-**
Test: build clang9_debug

### CODE REVIEW

File: OrbitLinuxTracing/LibunwindstackUnwinder.cpp [hunk 1]
```diff
@@ -11,6 +11,10 @@
 #include <array>
 #include <cstdint>
 
+#ifndef NDEBUG
+#include "OrbitBase/Logging.h"
```
Comment (dimitry-) [inline OrbitLinuxTracing/LibunwindstackUnwinder.cpp:15]: without it compilation fails on L56

Comment (dpallotti) [reply OrbitLinuxTracing/LibunwindstackUnwinder.cpp:15]: Do we upset IWYU if we include it regardless of `NDEBUG`?

**Review by dpallotti — APPROVED** [review]

File: OrbitLinuxTracing/LibunwindstackUnwinder.cpp (hunk 1 above)
Comment (beckerhe) [reply OrbitLinuxTracing/LibunwindstackUnwinder.cpp:15]: Ah makes sense. Can you also add `// IWYU pragma: keep` behind the include?

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM. Thanks!

File: OrbitLinuxTracing/LibunwindstackUnwinder.cpp (hunk 1 above)
Comment (dimitry-) [reply OrbitLinuxTracing/LibunwindstackUnwinder.cpp:15]: done.

Comment (beckerhe) [reply OrbitLinuxTracing/LibunwindstackUnwinder.cpp:15]: I think it has to go on the same line as the include itself. Ref: https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/IWYUPragmas.md

Comment (dimitry-) [reply OrbitLinuxTracing/LibunwindstackUnwinder.cpp]: done.

**Pull request merged**